engine/src/flutter/lib/ui/platform_dispatcher.dart

```diff
@@ -305,6 +305,13 @@ class PlatformDispatcher {
     _invoke1<ViewFocusEvent>(onViewFocusChange, _onViewFocusChangeZone, event);
   }
 
+  /// Opaque engine identifier for the engine running current isolate. Can be used
+  /// in native code to retrieve the engine instance.
+  /// The identifier is valid while the isolate is running.
```

Comment:
It's not immediately obvious when `engineId` is null. Is that if the platform doesn't support engine IDs? Or is this null when the Dart isolate isn't running? Or during startup? This might be worth a quick explanation in the public docs.